pom.xml

```diff
@@ -472,22 +472,42 @@
         </dependencies>
         <!-- Specifying configuration here will take effect on the execution of "mvn checkstyle:checkstyle",
              but will not take effect on the execution of "mvn site"  -->
-        <configuration>
-          <configLocation>${project.basedir}/config/checkstyle_sevntu_checks.xml</configLocation>
-          <failOnViolation>true</failOnViolation>
-          <includeResources>false</includeResources>
-          <includeTestResources>false</includeTestResources>
-          <logViolationsToConsole>true</logViolationsToConsole>
-          <maxAllowedViolations>0</maxAllowedViolations>
-          <violationSeverity>error</violationSeverity>
-          <propertyExpansion>project.basedir=${project.basedir}</propertyExpansion>
-          <sourceDirectory>${project.basedir}/src</sourceDirectory>
-          <excludes>**/it/resources/**/*,**/test/resources/**/*,**/test/resources-noncompilable/**/*</excludes>
-        </configuration>
         <executions>
           <execution>
             <id>sevntu-checkstyle-check</id>
             <phase>verify</phase>
+            <configuration>
+              <configLocation>${project.basedir}/config/checkstyle_sevntu_checks.xml</configLocation>
+              <failOnViolation>true</failOnViolation>
+              <includeResources>false</includeResources>
+              <includeTestResources>false</includeTestResources>
+              <logViolationsToConsole>true</logViolationsToConsole>
+              <maxAllowedViolations>0</maxAllowedViolations>
+              <violationSeverity>error</violationSeverity>
+              <propertyExpansion>project.basedir=${project.basedir}</propertyExpansion>
+              <sourceDirectory>${project.basedir}/src</sourceDirectory>
+              <excludes>**/it/resources/**/*,**/test/resources/**/*,**/test/resources-noncompilable/**/*</excludes>
```

Comment:
You configured Checker to process all files.
But you do not process all files from repository, you do this only under "src" folder of repo.
please extend config to process all files in git repository (yml, sh, .......) except for ".git" folder.